### ISSUE TITLE
[Accessibility] 3xVery low contrast, 1xMissing form label and 4xMissing alternative text - Admin Panel page

### ISSUE THREAD

**Issue by cmunteanuRW** [Talleh/VolleyballPub#6]
**Steps to reproduce:**

https://github.com/user-attachments/assets/a113a7d3-c205-4ad0-9c11-d78c52b063a2

**Actual Result:**

![Image](https://github.com/user-attachments/assets/0d1ff195-db71-4c52-a246-43531957fbba)

**Expected Result:**
No accessibility issues are present
